lm/lm_image.py

```diff
@@ -0,0 +1,120 @@
+__all__ = ["LmImageJob"]
+
+import shutil
+
+from sisyphus import *
+from i6_core import rasr
+from i6_core import util
+
+Path = setup_path(__package__)
+
+
+class LmImageJob(rasr.RasrCommand, Job):
+    """
+    pre-compute LM image without generating global cache
+    """
+
+    def __init__(
+        self,
+        crp,
+        extra_config=None,
+        extra_post_config=None,
+        encoding="utf-8",
+        renormalize=False,
+    ):
+        kwargs = locals()
+        del kwargs["self"]
+
+        self.text_file = text_file
```

Comment:
`text_file` is not defined? Is this a missing argument for `__init__`?

(edit:) Does not seem to be used. Can this line be deleted?

Reply:
Yes, this should be deleted, sorry